WordPress/Classes/Services/BlogService.m

```diff
@@ -305,40 +310,74 @@ - (void)syncBlogStaggeringRequests:(Blog *)blog
 - (void)staggerSyncPostsForBlog:(Blog *)blog
 {
     PostService *postService = [[PostService alloc] initWithManagedObjectContext:self.managedObjectContext];
-    [postService syncPostsOfType:PostServiceTypePost forBlog:blog success:^{
+    NSManagedObjectID *blogID = blog.objectID;
+    void(^next)() = ^ {
+        Blog *blog = (Blog *)[self.managedObjectContext existingObjectWithID:blogID error:nil];
```

Comment:
Again we are ignoring the possible error returned we should at least log it. This happens on other calls bellow, so I will not state it again but I think we should change them all to track a possible error and log it.